src/components/bookList.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import DisplayBooks from './displaybooks';
+
+export default function BooksList() {
+  // eslint-disable-next-line no-unused-vars
+  const [books, setBooks] = useState([
```

Comment:
- [ ] Kindly remove the `eslint` disabling comment at the beginning of this file. It is not good JavaScript practice to use eslint disabling comments. `Eslint` helps developers write clean and industry-standard JavaScript code. 👍

Reply:
- Hey, please, as suggested by the previous reviewer, kindly remove the `eslint` disable comment here as he's stated the reason above. Please, kindly work on this issue :pray: